st2common/st2common/constants/action.py

```diff
@@ -57,6 +59,7 @@
     LIVEACTION_STATUS_RUNNING,
     LIVEACTION_STATUS_SUCCEEDED,
     LIVEACTION_STATUS_FAILED,
+    LIVEACTION_STATUS_CANCELING,
```

Comment:
since you added this state I think the is_action_canceled method should become is_action_canceled_or_canceling